README.md

```diff
@@ -139,6 +139,58 @@ example:
 
 A SessionID remains valid for 24 hours.
 
+#### Logout from Session
+
+New sessionID will be generated for each task run from Ansible collection SDK if a sessionID is not passed as param to task. And the session will be logged out once task is done.
+
+For example, In the below task, session will be logged out once it is done.
```

Comment:
Scenario 1: In the below task, session will be logged out once it is done. So if we run multiple tasks then in no condition multiple sessions remain active.

Reply:
Changed to this.